src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj

```diff
@@ -88,6 +89,16 @@
       <LayoutFile Include="@(PackFile)">
         <TargetPath>..\NuGet.Build.Tasks.Pack\%(PackFile.RecursiveDir)%(PackFile.Filename)%(PackFile.Extension)</TargetPath>
       </LayoutFile>
+
+      <!-- Include some of the Sdks from the Stage 0 CLI for performance tests-->
```

Comment:
Are these the SDK files that the CLI requires that are not produced by dotnet/sdk?

Reply:
These are SDK files that the Performance tests require that are not produced by dotnet/sdk